tests/e2e/pact/broker/verify.ts

```diff
@@ -93,40 +87,43 @@ async function verifyConvert(): Promise<any> {
 
 /* eslint-disable  @typescript-eslint/no-explicit-any */
 async function verifyPrepare(): Promise<any> {
-  endpoint = "prepare"
-  pactGroup = ""
-  resetBackOffRetryTimer()
-  await verifyWith2Retries()
-}
-
-function isSandbox() {
-  return process.env.PACT_PROVIDER_URL.includes("sandbox")
+    await getPreparePactGroups().reduce(async (promise, group) => {
+      await promise
+      endpoint = "prepare"
```

Comment:
Is there a reason for `endpoint` and `pactGroup` to be top level variables instead of just being passed into `verifyX()`?

Reply:
Leftover from refactoring, changed to pass variables in